<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XRtmpStreamerClass</class>
 <widget class="QWidget" name="XRtmpStreamerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XRtmpStreamer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
color:rgb(255,255,255);
font:75 12pt &quot;黑体&quot;;
}
QWidget
{
background-color:#383838;
}
QPushButton:hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
 border: none;
border-radius:15px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(39, 39, 39, 255), stop:0.495 rgba(120, 120, 120, 255), stop:0.505 rgba(120,120, 120, 255), stop:1 rgba(39, 39,39, 255));
 border: none;
border-radius:15px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }

QLineEdit
{
border-radius:10px;
background-color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;

}
QComboBox
{
background-color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;

}</string>
  </property>
  <widget class="QLineEdit" name="outUrl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>rtmp://192.168.56.1:1935/live/myStream</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>推流地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>430</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>输入RTSP URL或
摄像头编号0、1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inUrl">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>490</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始直播</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>美颜级别</string>
   </property>
  </widget>
  <widget class="QComboBox" name="face">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>一级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>二级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三级</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择视频文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>390</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>推送视频</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>140</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ImageView"/>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;综合课程设计3-新媒体数据发布直播平台&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xrtmpstreamer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>XRtmpStreamerClass</receiver>
   <slot>Stream()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>841</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>XRtmpStreamerClass</receiver>
   <slot>Stream_mp4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>833</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>XRtmpStreamerClass</receiver>
   <slot>push_mp4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>799</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Stream()</slot>
  <slot>Stream_mp4()</slot>
  <slot>push_mp4()</slot>
 </slots>
</ui>
